models/classes/menu/SectionVisibilityFilter.php

```diff
@@ -73,26 +70,43 @@ public function isVisible(string $section): bool
             $userSettings->getSetting(
                 UserSettingsInterface::INTERFACE_MODE
             ) === GenerisRdf::PROPERTY_USER_INTERFACE_MODE_SIMPLE
-            && in_array($section, self::SIMPLE_INTERFACE_MODE_HIDDEN_SECTIONS)
+            && in_array($sectionPath, self::SIMPLE_INTERFACE_MODE_HIDDEN_SECTIONS)
```

Comment:
Is it possible to use strict comparison here?
```suggestion
            && in_array($sectionPath, self::SIMPLE_INTERFACE_MODE_HIDDEN_SECTIONS, true)
```

Reply:
I will, but this is the previous behavior, nothing changed here